projects/arlas-toolkit/src/lib/components/permissions-creator/permissions-creator-dialog/permissions-creator-dialog.component.html

```diff
@@ -0,0 +1,42 @@
+<div mat-dialog-title class="title">{{'Create permission' | translate}}</div>
+<mat-dialog-content class="content">
+    <div class="announcement"  *ngIf="!descriptionInputHidden">
+        <span>{{'The current filters applied to' | translate}} </span>
+        <span class="collection">{{data.mainCollection}}, </span>
+        <span>{{'will be associated to a' | translate}} </span>
+        <span>{{'new permission' | translate}}.</span>
```

Comment:
i think this key and the previous one should be associated in case a language does not have the same sentence construction